### DISCUSSION TITLE
App v1.0.0-2

### DISCUSSION THREAD

**Discussion by 4gray** [4gray/iptvnator#467] [General]
See the assets to download this version and install.

Changes:
* [feat(player): add volume control and persistence for video players](https://github.com/4gray/iptvnator/commit/67620f065409d60ec08216d29e2acefa462a45ca)
* [feat: add update mechanism](https://github.com/4gray/iptvnator/commit/94ef51d0ff0275256440659dc83e5db4a78a64cc)
* [style: visual improvements on settings page](https://github.com/4gray/iptvnator/commit/737d4be3872fad35bbd86ddd19e2002fe423c24c)
* [refactor: remove outdated dependencies](https://github.com/4gray/iptvnator/commit/58306c9a0808351dbd8180b485c0f530653f233b)
* [refactor(angular): switch to standalone components API](https://github.com/4gray/iptvnator/commit/3d4f549eefe68177ef4d90ae57b662d04bb88670)

**Full Changelog**: https://github.com/4gray/iptvnator/compare/app-v1.0.0-1...app-v1.0.0-2

<hr /><em>This discussion was created from the release <a href='https://github.com/4gray/iptvnator/releases/tag/app-v1.0.0-2'>App v1.0.0-2</a>.</em>

**Comment by nt74** [comment]
Is it possible to update all json config files on the release to be on-par with the Git release?
They have the old 0.16.0 version files on them.

**Reply by 4gray** [reply]
Which json files exactly?

**Reply by nt74** [reply]
The important ones would be: `package.json`, `package-lock.json`. But I have included a diff file for you to analyse.
[diff.txt](https://github.com/user-attachments/files/17930540/diff.txt)

**Reply by 4gray** [reply]
`package.json` and `package-lock.json` are up-to-date 
https://github.com/4gray/iptvnator/blob/electron/package.json
https://github.com/4gray/iptvnator/blob/electron/package-lock.json

**Reply by nt74** [reply]
Yes on the Git release, on the Release you get the diffs I sent to you.
Download this https://github.com/4gray/iptvnator/archive/refs/tags/app-v1.0.0-2.tar.gz
And compare the files once again.

For package maintainers like myself, We like to lock the upstream to the latest release not the latest Git upstream.

**Reply by 4gray** [reply]
Ahh, got it, looks like I updated them after the release. Thanks for the hint!
